client/src/app/pages/applications/components/application-form/application-form.tsx

```diff
@@ -525,6 +525,7 @@ export const ApplicationForm: React.FC<ApplicationFormProps> = ({
                   const selectionValue = selection as OptionWithValue<string>;
                   onChange(selectionValue.value);
                 }}
+                onBlur={onChange}
```

Comment:
Did you mean onBlur here?

Reply:
Yes. That works around the issue but maybe there is better way.